goth_tests/e2e/vm/test_e2e_rule_partner_outbound.py

```diff
@@ -41,7 +41,26 @@ async def test_e2e_rule_partner_outbound(
         {"name": "requestor", "type": "Requestor", "address": "d1d84f0e28d6fedf03c73151f98df95139700aa7" },
         {"name": "provider-1", "type": "VM-Wasm-Provider", "address": "63fc2ad3d021a4d7e64323529a55a9442c444da0", "use-proxy": True},
     ]
+
+    assets_root = Path(__file__).parent / "assets";
+    node_types = [
```

Comment:
Could you please add comments about why it is needed? Here and in other e2e test so "knowledge" will stay with us.

Reply:
this is temporary, will remove it once the new goth default assets are published